keps/sig-cloud-provider/2019-01-25-out-of-tree-azure.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Support Out-of-Tree Azure Cloud Provider
+authors:
+  - @andrewsykim
```

Comment:
cc @justaugustus @khenidak to update SIG Azure KEP once this PR is merged

Reply:
should this file put under sig-azure directory?

Reply:
@andrewsykim @feiskyer -- I think when someone picks one of the templates up, they can maybe:
- Move the template into the relevant SIG directory
- Update the [top-level out-of-tree KEP}(https://github.com/kubernetes/enhancements/blob/master/keps/sig-cloud-provider/2019-01-25-removing-in-tree-providers.md) to include a link to the SIG-specific one (in the same PR)

Reply:
sure, make sense

Reply:
yeah that works for me, will leave it the file here for now in this PR then :)